demo/server/config-dev.js

```diff
@@ -0,0 +1,77 @@
+
+
+// Server side application configuration
+var config = {
+  // Application port number
+  "port": process.env.PORT || process.env.FH_PORT || 8001,
+  "morganOptions": "dev",
+  "logStackTraces": true,
+  // Use this parameter to disable inserting demo data into mongodb
+  "seedDemoData": true,
+  // General security configuration
+  "security": {
+    "adminRole": "admin",
+    "userRole": "user",
+    // Passport.js security configuration
+    "passportjs": {
+      // Passport.js secret for JWT tokens. Provide custom value even for development needs
+      "jwtSecret": "D837131FD17F62CB85FBD5919563086369691F4D42379C3596F811839A8992CBA1FBA88DF243BF2481940F112D339C33283BDFEF29A13612550EDAAAB7B5E061",
+      // Allows to provide custom login page messages
+      "portalLoginPage": {
+        "title": "RainCatcher Portal",
+        "invalidMessage": "Invalid Credentials"
+      },
+    },
+    // Keycloak configuration. Uncomment to enable keycloak integration
+    // "keycloak": {
```

Comment:
Do you think it would be best to fill in the details here? Otherwise we can add a note that they need to fill these in themselves? 

```
   "realm": "raincatcher",
   "bearer-only": true,
   "auth-server-url": "http://localhost:8080/auth",
   "ssl-required": "external",
   "resource": "raincatcher-cloud",
   "use-resource-role-mappings": true
```

Reply:
hmm.. I think we will cover that in documentation. Missing details means some action. If we provide some values some people may take them for granted and then file bugs that stuff do not work.